packages/sources/css/src/components/link/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "@vtmn/css-link",
+  "version": "0.1.0",
```

Because lerna will by itself. You need to start at 0.0.0.
```suggestion
  "version": "0.0.0",
```

---

OK